i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md

```diff
@@ -10,16 +10,16 @@ O APM (monitoramento de desempenho de aplicação) refere-se a uma família de p
 
 ### Exemplo de APM - um produto comercial que visualiza o desempenho de aplicações de serviço cruzado
 
-![Exemplo de APM](/assets/images/apm1.png "Exemplo de APM")
+![Exemplo de APM](./assets/images/apm1.png)

```

Comment:
accidentally removed

Reply:
Has this happened due to the absolute path converting script?
Should I rollback everything?

Reply:
I think I'll build a migration script that gonna use markdown parsing which is more reliable than my replace script

Reply:
@anediaz I've created the repo for the migration script and send you an invite in the email to be a collaborator.

The repo is: [rluvaton/migrate-nodebestpractices-to-docusaurus](https://github.com/rluvaton/migrate-nodebestpractices-to-docusaurus)

I've explained everything there, here's the [link](https://github.com/rluvaton/migrate-nodebestpractices-to-docusaurus/discussions/1#discussion-3425263)

Reply:
I've just accepted your invitation!
I could take a look to your repo tomorrow, this is an excellent initiative!

Thank you once again Raz!